### ISSUE TITLE
pushover: possibility to use priority argument

### ISSUE THREAD

**Issue by vmpr** [kha7iq/pingme#72]
Hi and thanks for your great tool!
I would like to be able to use the priority argument with your pingme tool for pushover, the go wrapper you are using ( https://github.com/gregdel/pushover ) has it already built in, so it should be not a significant effort to implement it into your code, but I can't program go so its hard for me to implement it myself :(
thank you and have a great day!
cheers Ringo

**Comment by kha7iq** [comment]
Hello @vmpr 
I saw its is available in the wrapper i am using, allow me some time to look into the implementation

**Comment by kha7iq** [comment]
Hello @vmpr 
I have added the priority flag for pushover, can you build from [priority-for-pushover](https://github.com/kha7iq/pingme/tree/priority-for-pushover) branch  and test it. 
1. clone the repo & branch
 ```bash
git clone -b priority-for-pushover https://github.com/kha7iq/pingme.git
```
2. cd into the folder
3.  build the binary
```bash
go build -o pingme main.go
```
4. Send a message 
```bash
./pingme pushover --token abc --msg "Priority value 1 " --title "Notify" -u xxxxxxx  -p 1
```

Let me know if this works for you , i will merge it after to main after that.

**Comment by vmpr** [comment]
Hello @kha7iq 

I've tested it and it works like a charm! thank you!

cheers Ringo

**Comment by kha7iq** [comment]
Thank you for testing.
Changes merged https://github.com/kha7iq/pingme/pull/74
Available in release 0.2.6
